pyproject.toml

```diff
@@ -31,9 +31,9 @@ dependencies = [
     "protobuf>=3.12.2",  # minimum required based on latest ansys-grpc-mapdl
     "psutil>=5.9.4",
     "pyansys-tools-versioning>=0.3.3",
-    "ansys-tools-path>=0.2.0",
+    "ansys-tools-path>=0.2.4",
     "pyiges>=0.1.4",
-    "pyvista>=0.33.0",
+    "pyvista>=0.33.0<0.39.0",
```

Comment:
The recommendation from PyVista is to use the vtk module directly as an import... Why not change the locations where it complains inside the library? We've done this in other PyAnsys packages. Setting an upper-limit on PyVista is definitely not recommended

Reply:
I will investigate that. Thank you @RobPasMue

Reply:
The key point is changing whatever m``pyvista._vtk`` import for ``vtk`` directly =)